packages/analytics/amplify_analytics_pinpoint_dart/lib/src/analytics_plugin_impl.dart

```diff
@@ -323,4 +321,53 @@ class AmplifyAnalyticsPinpointDart extends AnalyticsPluginInterface {
     await _eventClient.close();
     await _eventStorageAdapter.close();
   }
+
+  @visibleForTesting
+  /// Retrieve the stored pinpoint endpoint id
```

Comment:
Annotations should always be directly above the function, member, or class that they annotate. That means that they should be below doc comments.

This is likely failing a formatting check. If you run formatting on this file, a space will be added below the annotation because the formatter doesn't recognize what it is annotating.

```suggestion
  /// Retrieve the stored pinpoint endpoint id
  @visibleForTesting
```